app/views/teachers/learning_resources/_learning_resource.html.erb

```diff
@@ -8,24 +8,24 @@
         </strong>
       </p>
       <p>
-        <strong>Description:</strong>
+        <strong>Descripción:</strong>
```

Comment:
You forgot the word 'name' in the line 7 of this file

Reply:
No sean cabrones use I18n
```erb
<strong><%= t('.description')%</strong>
```
https://guides.rubyonrails.org/i18n.html

Y ya saquenme de aqui @bryanmg, diario me llegan un chingo de notificaciones. les desactive los github actions btw jjjjjj.
Saludos :heart: